pkgs/applications/audio/indicator-sound-switcher/default.nix

```diff
@@ -0,0 +1,65 @@
+{ python3Packages
+, lib
+, fetchurl
+, perlPackages
+, gettext
+, gtk3
+, gobject-introspection
+, intltool, wrapGAppsHook, glib
+, librsvg
+, libayatana-appindicator-gtk3
+, libpulseaudio
+, keybinder3
+, gdk-pixbuf
+}:
+
+let
+  name = "indicator-sound-switcher";
+  version = "2.3.6";
+in
+python3Packages.buildPythonApplication {
+#python3Packages.buildPythonPackage {
+  pname = name;
+  inherit version;
```

Comment:
```suggestion
python3Packages.buildPythonApplication rec {
  pname = "indicator-sound-switcher";
  version = "2.3.6";
```

Reply:
Do we really need to use `rec`?

Reply:
Yes, because you're referencing the version attribute in the fetchFromGitHub one.